app/services/link_checker_service.rb

```diff
@@ -8,24 +8,26 @@ def initialize(link)
   def run
     begin
       check_link
-    rescue RestClient::ExceptionWithResponse
-      link.update(broken: true)
-      create_broken_link_task
+    rescue RestClient::ExceptionWithResponse => error
+      link.broken = true
+      link.save(validate: false)
```

Comment:
Why do we need to circumvent the validation? Could we be smarter with our validation rules instead?

Reply:
@andrewgarner as mentioned face to face, I think this is part of a wider issue, so I suggest we deal with this in a separate PR.

Reply:
👍